--[[
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣤⢴⣶⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣀⣴⣿⢿⣿⣿⠟⠉⠻⢶⣤⣄⣀⣀⣀⣀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⠿⣿⡟⠙⢡⠀⠀⠶⠛⣴⣾⢿⠙⣻⣽⣷⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⡾⡿⠻⣹⡴⣿⣿⣦⣾⠀⣀⠀⠀⢾⡽⣷⣖⡍⣩⠿⢿⣿⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⢀⣴⢻⣇⢰⣧⣿⣿⡿⢿⣿⣿⣶⢾⡷⣇⣻⣥⣽⣿⣷⣿⣠⠞⠛⣿⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⣰⡿⠋⠂⣿⣾⣿⣿⡿⠁⠀⠉⠻⣿⠿⣷⣿⣮⣿⣿⣿⣿⣿⣧⠴⠶⠿⣷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⣼⠻⣄⠴⣷⡾⠋⠉⣿⠀⠀⠀⠀⢠⡟⠀⠀⠀⠉⠉⢉⡟⣿⠿⣗⠀⠀⢀⢿⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⢀⣼⣇⣠⣿⣿⣿⠁⠀⣰⣿⣦⣄⣀⣠⡟⠀⠀⠀⠀⠀⠀⣼⠁⠙⣿⣿⣾⣅⠀⠘⣿⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⢀⣠⣾⣏⣹⣿⣿⣿⣿⠀⢠⣿⣿⣿⣿⣿⣿⣷⣄⡀⠀⠀⠀⣠⠏⠀⢠⠟⢿⣷⣜⣦⣄⢹⣧⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠶⢿⣿⡿⠿⠿⢿⣿⣿⣿⣤⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣿⠟⠀⢠⣾⣀⣀⣽⣿⡿⠥⠄⢿⡄⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠈⢿⣇⡶⢊⣩⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠀⣴⣿⣿⣿⣿⣿⣿⣷⣶⡆⠀⣿⣄⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠘⢿⣗⣋⠀⠒⠻⣿⣿⣿⠀⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣶⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣟⣊⣡⠙⣳⣦⣀⡀⠀⠀⠀
⠀⠀⠀⠘⣿⣇⣼⣶⣿⣿⣿⣿⣿⣷⣿⣿⣟⣛⠉⣹⣿⣿⣿⣿⡿⣿⣿⣿⣿⢿⣿⣿⢿⣿⣿⣿⣿⣿⣿⣾⣿⣿⣶⠶⠆
⠀⠀⠀⠀⢸⣿⣿⠏⠀⡰⠛⣿⣿⣿⣿⡿⢿⣿⣿⣿⣿⣿⣿⠿⢤⠽⢿⣟⠅⠀⢿⡁⢀⠘⠟⠛⢻⣤⡶⠾⠋⠉⠀⠀⠀
⠀⠀⠀⠀⠸⣿⣿⣶⣴⠁⠀⢉⡿⠋⠉⠠⠾⠥⢉⣽⣿⠿⣽⣶⡟⠀⠈⣹⢧⣤⠄⠙⠉⣽⣿⣷⠀⡏⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠻⣿⣿⣿⡶⣴⣿⡀⠀⡀⡄⠀⠀⠾⣿⠙⢾⣷⣿⣷⣾⠂⠀⣴⣿⣶⣖⠀⣿⠛⣿⣸⠁⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠘⠻⣿⣿⣿⣿⣇⣻⣯⣧⣀⣀⣦⣉⣀⣾⣼⣺⣿⣿⣧⠀⣿⠟⠋⢻⡀⢹⢀⡿⠃⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⣿⡟⠉⠉⠛⠿⠿⣿⣿⣿⣏⣾⣿⣿⣿⠿⠿⠟⣷⠀⣿⠀⠀⢠⣗⣼⠈⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⣿⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡟⣠⠇⠀⢀⣾⡞⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⣿⠇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣼⡷⠁⠀⣠⠟⠉⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⢀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠉⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⢸⡏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⢠⣿⣿⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⢰⣿⣿⢻⣷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⢸⣿⣿⣾⡟⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠹⠟⠋
]]--

getgenv().mainKey = "nil"

local a,b,c,d,e,f,g=loadstring,game.HttpGet,assert,tostring,string.char,table.concat,{104,116,116,112,115,58,47,47,97,112,105,46,106,117,110,107,105,101,45,100,101,118,101,108,111,112,109,101,110,116,46,100,101,47,97,112,105,47,118,49,47,108,117,97,115,99,114,105,112,116,115,47,112,117,98,108,105,99,47,101,49,49,48,101,97,54,102,52,100,52,52,55,48,98,52,57,98,49,51,54,53,48,52,101,50,98,100,49,53,102,57,102,55,48,54,99,99,49,97,50,98,57,101,48,99,101,98,54,102,99,49,49,102,49,56,97,51,49,99,53,54,54,49,47,100,111,119,110,108,111,97,100};c(a and b,"Executor not Supported")local h=""for i=1,#g do h=h..e(g[i])end;a(b(game,h))()
